src/fparser/two/tests/test_bases.py

```diff
@@ -114,3 +114,25 @@ def test_io_ctrl_spec_errors():
     # description
     obj = Io_Control_Spec.match("not_unit=23")
     assert obj is None
+
+
+def test_blockbase_tofortran_non_ascii(f2003_create):
```

Comment:
Make use of `@pytest.mark.usefixtures("f2003_create")` here to be consistent?

Reply:
Yes. In fact I've done throughout and removed the old-style creation of a parser object from the start of the file.